vertical-pod-autoscaler/pkg/admission-controller/config.go

```diff
@@ -181,3 +186,28 @@ func selfRegistration(clientset kubernetes.Interface, caCert []byte, webHookDela
 		klog.V(3).Info("Self registration as MutatingWebhook succeeded.")
 	}
 }
+
+// ConvertLabelsToMap convert the labels from string to map
+// the valid labels format is "key1:value1,key2:value2", which could be converted to
+// {"key1": "value1", "key2": "value2"}
+func ConvertLabelsToMap(labels string) (map[string]string, error) {
```

Comment:
```suggestion
// convertLabelsToMap convert the labels from string to map
// the valid labels format is "key1:value1,key2:value2", which could be converted to
// {"key1": "value1", "key2": "value2"}
func convertLabelsToMap(labels string) (map[string]string, error) {
```

I think this can be private?

Reply:
Can you add a unit test for this function?

Reply:
sure, add ut and set it private